ampersand-state.js

```diff
@@ -756,11 +760,11 @@ var dataTypes = {
             // if this has changed we want to also handle
             // event propagation
             if (previousVal) {
-                this.stopListening(previousVal);
+                this.stopListening(previousVal, 'all', this._getCachedEventBubblingHandler(attributeName));
```

Comment:
Should we remove the cached version from `this._eventBubblingHandlerCache` once it's unbound so it doesn't stay in memory?